src/components/footer/Footer.tsx

```diff
@@ -72,8 +72,11 @@ function Footer({ navigationItems }: Props) {
         copyrightHolder={t("copyright_holder")}
         copyrightText={t("copyright_text")}
       >
-        <HDSFooter.Item href="#" label={t("about_us")} />
-        <HDSFooter.Item href="#" label={t("accessibility_statement")} />
+        <HDSFooter.Item href="/about" label={t("about_us")} />
+        <HDSFooter.Item
+          href="/accessibility"
+          label={t("accessibility_statement")}
+        />
```

Comment:
Using `as={LInk}` would translate the paths they point to (which you configured in the `i18nRoutes.config.js` file).